providers/implementations/keymgmt/ec_kmgmt.c

```diff
@@ -317,27 +284,24 @@ int ec_import(void *keydata, int selection, const OSSL_PARAM params[])
     /*
      * In this implementation, we can export/import only keydata in the
      * following combinations:
-     *   - domain parameters only
+     *   - domain parameters (+optional other params)
      *   - public key with associated domain parameters (+optional other params)
      *   - private key with associated public key and domain parameters
      *         (+optional other params)
      *
      * This means:
      *   - domain parameters must always be requested
      *   - private key must be requested alongside public key
-     *   - other parameters must be requested only alongside a key
+     *   - other parameters are always optional
```

Comment:
This has changed because fromdata requires params+other.

Reply:
... and yet, there's still a separate call of `ec_key_otherparams_fromdata()`, which is still unchanged, so the code here doesn't match what you say, and I'm not sure I understand why that condition had to be removed.

Reply:
What I mean is..
The function EVP_PKEY_fromdata()

does this
```
    if (ctx->operation == EVP_PKEY_OP_PARAMFROMDATA)
        selection = OSSL_KEYMGMT_SELECT_ALL_PARAMETERS;
```
Which will cause an issue

Reply:
So we either
(1) relax the constraints as I have - or 
(2) we have yet another API that does domain params only.. 
      We have EVP_PKEY_param_fromdata_init & EVP_PKEY_key_fromdata_init currently.. OR
(3) we have one API that allows you to specify the selection instead.

Reply:
Ah, gotcha. I think that what you've done is sufficient for now. Thanks for elaborating.